examples/jenkins/pipeline/samplepipeline.yaml

```diff
@@ -0,0 +1,327 @@
+apiVersion: v1
```

Comment:
did you convert samplepipeline.json to yaml?  if so, let's delete the json one.  (might need to make sure no tests are referencing it)